pkg/providers/amifamily/ami.go

```diff
@@ -79,25 +81,25 @@ func (a AMIs) Sort() {
 	})
 }
 
-func (a AMIs) String() string {
+func String(amis []v1beta1.AMI) string {
 	var sb strings.Builder
-	ids := lo.Map(a, func(a AMI, _ int) string { return a.AmiID })
-	if len(a) > 25 {
+	ids := lo.Map(amis, func(a v1beta1.AMI, _ int) string { return a.ID })
```

Comment:
What's the point of changing it from this type to the v1beta1 AMI type?

Reply:
External we will only be using v1beta1 for formatting

Reply:
Do we need this AMI struct anymore then? Seems like we're doing some translation back and forth, where the only difference is the CreationTimestamp. I'm happy to punt to another PR, but think it's worth refactoring eventually.